Lesson6/gokhan/index.html

```diff
@@ -0,0 +1,159 @@
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Pricing example for Bootstrap</title>
+		<link rel="stylesheet" href="./styles.css" />
+	</head>
+	<body>
+		<nav>
+			<a href="">Company name</a>
+			<ul class="nav-list">
+				<li><a href="">Features</a></li>
+				<li><a href="">Enterprise</a></li>
+				<li><a href="">Support</a></li>
+				<li><a href="">Pricing</a></li>
+				<li><a href="" class="btn1 btn-outlined">Sign up</a></li>
+			</ul>
+		</nav>
+
+		<main>
+			<div class="hero-content">
+				<h1>Pricing</h1>
+				<p>
+					Quickly build an effective pricing table for your potential customers
+					with this Bootstrap example. It's built with default Bootstrap
+					components and utilities with little customization.
+				</p>
+			</div>
+			<div class="cards-container">
+				<div class="card">
+					<h2>Free</h2>
+					<div class="card-content">
+						<span>
+              <span class="price">$0</span>&nbsp<span class="interval">/&nbspmo</span>
+            </span>
+						<ul>
+							<li>10 users included</li>
+							<li>2 GB of storage</li>
+							<li>Email support</li>
+							<li>Help center access</li>
+						</ul>
+						<a class="btn2 btn-outlined" href="">Sign up For Free</a>
+					</div>
+				</div>
+				<div class="card">
+        <h2>Pro</h2>
+					<div class="card-content">
+						<span>
+              <span class="price">$15</span>&nbsp<span class="interval">/&nbspmo</span>
+            </span>
+						<ul>
+							<li>20 users included</li>
+							<li>10 GB of storage</li>
+							<li>Priority email support</li>
+							<li>Help center access</li>
+						</ul>
+						<a class="btn3 btn-outlined" href="">Get Started</a>
+          </div>
+        </div>
+				<div class="card">
+          <h2>Enterprise</h2>
+					<div class="card-content">
+						<span>
+              <span class="price">$29</span>&nbsp<span class="interval">/&nbspmo</span>
+            </span>
+						<ul>
+							<li>30 users included</li>
+							<li>15 GB of storage</li>
+							<li>Phone and  email support</li>
+							<li>Help center access</li>
+						</ul>
+						<a class="btn4 btn-outlined" href="">Contact Us</a>
+          </div>
+        </div>
+			</div>
+      <hr class="hr">
+      <div class="footers-container">
```

Comment:
Better to use semantic elements:
```suggestion
      <footer class="footers-container">
```

Reply:
Also, move the footer outside the <main></main> element